devsetup/roles/download_tools/tasks/main.yaml

```diff
@@ -97,14 +122,17 @@
     mode: '0755'
 
 - name: Link yq_linux_amd64 as yq
+  tags:
+    - yq
   ansible.builtin.file:
     src: "{{ lookup('env', 'HOME') }}/bin/yq_linux_amd64"
     dest: "{{ lookup('env', 'HOME') }}/bin/yq"
     state: link
 
 - name: Set proper golang on the system
   become: true
-  become_user: root
+  tags:
+    - goland
```

Comment:
`s/goland/golang`

Reply:
aren't we in the Land of Go? damned GPS :]